### PR TITLE
161211864 admin/attendant get specific salerecord

### PR DESCRIPTION
**Pull request by armstrongsouljah**
This PR seeks to add code  implementing the endpoints as specified.
I would like you honest review. Thank you.
@Rhyta @BarnaTB @codjoero @dannylwe @dxania@nearjay06 @irenyak1 @kdaglas@Dianawats@FahdJamy@mozele@mwinel And everybody else. Help a brother out.

### CODE REVIEW

**Comment by Rhytah** [comment]
Try to maintain to main criteria of route decorators as preferred by key stakeholders. 
i.e `/api/v1/user` . This helps simplify and express route creation

**Comment by armstrongsouljah** [comment]
okay, thank you.
let me refactor that. although am not sure which route in particular you meant.

**Comment by llwasampijja** [comment]
> This PR seeks to add code implementing the endpoints as specified.
> I would like you honest review. Thank you.
> @rhyta @BarnaTB @codjoero @dannylwe @dxania@nearjay06 @Irenyak1 @kdaglas@Dianawats@FahdJamy@mozele@mwinel And everybody else. Help a brother out.

This is good work

**Comment by llwasampijja** [comment]
> > This PR seeks to add code implementing the endpoints as specified.
> > I would like you honest review. Thank you.
> > @rhyta @BarnaTB @codjoero @dannylwe @dxania@nearjay06 @Irenyak1 @kdaglas@Dianawats@FahdJamy@mozele@mwinel And everybody else. Help a brother out.
> 
> This is good w

> > This PR seeks to add code implementing the endpoints as specified.
> > I would like you honest review. Thank you.
> > @rhyta @BarnaTB @codjoero @dannylwe @dxania@nearjay06 @Irenyak1 @kdaglas@Dianawats@FahdJamy@mozele@mwinel And everybody else. Help a brother out.
> 
> This is good work

Will tell you more when I take a look at your code!

File: .travis.yml [hunk 1]
```diff
@@ -2,9 +2,6 @@ language: python
 python:
   - "3.6"
 
-services:
-  - postgresql
```
Comment (kdaglas) [inline .travis.yml:6]: I don't think you need this for challenge 2 but its ok, thought to maybe remove redundant code then this leaves but don't worry it has no effect

File: app/tests/test_products.py [hunk 2]
```diff
@@ -63,39 +63,44 @@ def test_cannot_get_non_existent_id(self):
...
+                headers=headers,
+                data=json.dumps(self.sample_product)
+            )
         # print(res.data)
         print(res.status_code)
         self.product_obj.get_products()  
         self.assertEqual(500, res.status_code)
 
 
     def test_admin_dont_add_empty_product(self):
```
Comment (kdaglas) [inline app/tests/test_products.py:141]: This is really clean and reusable... great stuff

**Pull request merged**